README.adoc

```diff
@@ -323,6 +323,29 @@ foo&. bar
 foo&.bar
 ----
 
+=== Safe navigation
+
+Avoid long chains of `&.`. Replace with `.` if they introduce unnecessary conditional logic. 
```

Comment:
"long" sounds vague. How do you feel about removing it altogether?

Reply:
yes sure what alternative would you suggest after removing it?